### PR TITLE
docs: v4.0 release notes

### PR DESCRIPTION
**Pull request by adamewozniak**
## Description

v4.0 release notes :^) 

Will not merge until we have stakeholders agreed on Umee v4.0 & PF v2.0.3

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by toteki — APPROVED** [review]

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -6,64 +6,31 @@
...
+* See the [Leverage Module Spec](/x/leverage/README.md#historic-borrow-limit-value) for a description of how these prices are treated by the leverage protocol.
 
-Additional highlights:
-
-- Added `QueryMaxWithdraw` and `MsgMaxWithdraw` to allow user easily withdraw previously supplied tokens from the module back to the user balance.
-- Updated Cosmos SDK to v0.46.7
-
-Please see the [CHANGELOG](https://github.com/umee-network/umee/blob/v3.3.0/CHANGELOG.md) for an exhaustive list of changes.
-
-### Github Release
```
Comment (robert-zaremba) [inline RELEASE_NOTES.md:23]: let's keep the `Github Release` section. Better to be explicit and repeat informing users about the upgrade procedure, than going back and forth on a chat.

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -6,64 +6,31 @@
...
+Please see the [CHANGELOG](/CHANGELOG.md#v4.0.0) for an exhaustive list of changes.
 
 ### Update instructions
 
-- Note: Skip this step if you build binary from source and are able to properly link libwasmvm.
-  - Download `libwasmvm`:
-
-```bash
-$ wget https://raw.githubusercontent.com/CosmWasm/wasmvm/v1.1.1/internal/api/libwasmvm.$(uname -m).so -O /lib/libwasmvm.$(uname -m).so
-```
```
Comment (robert-zaremba) [inline RELEASE_NOTES.md:46]: Ditto, let's keep the libwasmvm info.

**Review by robert-zaremba — APPROVED** [review]
pre-approving. Let's keep information about github releases and libwasmv to make sure validators will have clear instructions.

File: RELEASE_NOTES.md (hunk 1 above)
Comment (adamewozniak) [reply RELEASE_NOTES.md:23]: f6cfe968cebd905d5ca1661832ea182d9d9bb05d

File: RELEASE_NOTES.md (hunk 2 above)
Comment (adamewozniak) [reply RELEASE_NOTES.md:46]: f6cfe968cebd905d5ca1661832ea182d9d9bb05d

**Comment by adamewozniak** [comment]
dead links are definitely not dead 🤔

**Pull request merged**